apps/portals/src/configurations/elportal/headerConfig.tsx

```diff
@@ -1,13 +1,36 @@
+import { AddAlertTwoTone } from '@mui/icons-material'
+import { Link } from '@mui/material'
 import React from 'react'
 import { HomePageHeaderConfig } from 'types/portal-config'
 
 const homePageHeader: HomePageHeaderConfig = {
   summary: (
     <>
-      Discover and explore exceptional longevity data, analyses, and tools generated by the translational longevity research projects supported by the National Institute on Aging (NIA).
+      Discover and explore exceptional longevity data, analyses, and tools
+      generated by the translational longevity research projects supported by
+      the National Institute on Aging (NIA).
+      <>
+        <Link
+          href="https://news.eliteportal.org/elite-portal-newsletter/"
+          target="_blank"
+          sx={{
+            color: 'white',
+            '&:hover': { color: 'white' },
+            '&:focus': { color: 'white' },
```

Comment:
Is the text in the header already white? You may be able to add the prop `variant='inherit'`. Not totally sure if that will work, since bootstrap 3 styles might still override the hover/focus selectors.

Reply:
The variant `inherit` did not fix the base color (which was still set to the default mui-based link primary color), or hover/focus state.